### PR TITLE
Added talk about panel properties. And apparently more stuff. wth git?

### PR DESCRIPTION
**Pull request by petterroea**

### CODE REVIEW

File: Fuse/04 - Layout.md [hunk 1]
```diff
@@ -29,6 +29,21 @@ The most basic type of panel is the `Panel`. Children of a Panel will be default
 
 Note that the element order in a `Panel` is the same as the layer order in popular graphics packages such as Photoshop; the layer that appears first in the UX-file will be layered on top of elements appearing later in the file.
 
+The `Panel` has a few properties you might be interrested in:
+
+* `Layout` - specifies a Layout for the `Panel`. A `StackPanel` is for example primarily just a `Panel` with a `StackLayout` controller.
```
Comment (sebbert) [inline Fuse/04 - Layout.md]: This is documented in greater detail [here](https://github.com/fusetools/handbook-docs/blob/5129d4a2a5be37f28319f419112acd73c95ad748/Fuse/04%20-%20Layout.md#the-layout-property). Possibly link to it or merge the two?

File: Fuse/04 - Layout.md [hunk 2]
```diff
@@ -29,6 +29,21 @@ The most basic type of panel is the `Panel`. Children of a Panel will be default
...
+* `Children` - contains a list of all child nodes of the `Panel`. This may for example come in handy when manipulating nodes with uno.
+
+> ### $(Layout:Setting the layout of a Panel manually)
```
Comment (sebbert) [inline Fuse/04 - Layout.md]: This is already documented [here](https://github.com/fusetools/handbook-docs/blob/5129d4a2a5be37f28319f419112acd73c95ad748/Fuse/04%20-%20Layout.md#the-layout-property)

File: Fuse/04 - Layout.md [hunk 3]
```diff
@@ -29,6 +29,21 @@ The most basic type of panel is the `Panel`. Children of a Panel will be default
 
 Note that the element order in a `Panel` is the same as the layer order in popular graphics packages such as Photoshop; the layer that appears first in the UX-file will be layered on top of elements appearing later in the file.
 
+The `Panel` has a few properties you might be interrested in:
```
Comment (sebbert) [inline Fuse/04 - Layout.md]: "Interrested" -> "Interested"

File: Fuse/04 - Layout.md [hunk 4]
```diff
@@ -29,6 +29,21 @@ The most basic type of panel is the `Panel`. Children of a Panel will be default
...
+
+* `Layout` - specifies a Layout for the `Panel`. A `StackPanel` is for example primarily just a `Panel` with a `StackLayout` controller. Check out @(The Layout property:layout) for more detail.
+* `Children` - contains a list of all child nodes of the `Panel`. This may for example come in handy when manipulating nodes with uno.
+
+> ### $(Layout:Setting the layout of a Panel manually)
```
Comment (sebbert) [inline Fuse/04 - Layout.md]: This whole section can be removed, it's already documented

File: Fuse/08 - Navigation.md [hunk 5]
```diff
@@ -77,10 +77,10 @@ We actually already built most of the behavior of the `PageControl` from scratch
 * `Active` - The currently active page
 * `CanGoBack` - `true` if you can navigate backwards
 * `CanGoForward` - `true` if you can navigate forward
-* `PageCount` - Current page number
+* `PageCount` - Number of pages
 * `PageProgress` - Progress in the navigation from 0.0 to the amount of pages minus one.
 
-One interresting feature is the fact that you can set the active `Page` in a `PageControl` by name using the `Active` property, referencing to the wanted `Page` by `Name`. In the following example, We have a 3-page app, and a button that allways returns the user to the first page.
+One interresting feature is the fact that you can set the active `Page` in a `PageControl` by name using the `Active` property, referencing to the wanted `Page` by `Name`. In the following example, We have a 3-page app, and a button that always returns the user to the first page.
```
Comment (sebbert) [inline Fuse/08 - Navigation.md]: interesting, not interresting

**Comment by petterroea** [comment]
It this cool?

**Comment by sebbert** [comment]
Nope, you didn't fix the typos.

**Comment by petterroea** [comment]
Ok, but now it's like, totally cool, right?

**Comment by sebbert** [comment]
Finally :smile:

**Pull request merged**